src/components/HorizontalRule/HorizontalRule.tsx

```diff
@@ -0,0 +1,42 @@
+// HorizontalRule
+import * as React from "react";
+import bem from "../../utils/bem";
+
+export interface HorizontalRuleProps {
+  /** Optional alignment value to align the horizontal rule to one side or the other when the width is less than 100%. If omitted, the horizobntal rule will have a default center alignment. */
+  align?: "left" | "right";
+  /** Additional attributes passed to the horizontal rule */
+  attributes?: { [key: string]: any };
+  /** ClassName you can add in addition to `horizontal-rule` */
+  className?: string;
```

Comment:
If one wants to add more than a single additional class, do they just pass all in a string?

Reply:
Yes.  

`className="classone classtwo"`